api/wordpress/_global/getPostTypeArchive.js

```diff
@@ -0,0 +1,87 @@
+import {initializeApollo} from '../connector'
+import queryPostsArchive from '../posts/queryPostsArchive'
+import {postTypes} from './postTypes'
+
+/**
+ * Retrieve post archive.
+ *
+ * @author WebDevStudios
+ * @param  {string} postType WP post type.
+ * @param  {string} orderBy  Order by: field.
+ * @param  {string} order    Order by: direction.
+ * @param  {string} cursor   Start/end cursor for pagination.
+ * @param  {bool}   getNext  Whether to retrieve next set of posts (true) or previous set (false).
+ * @param  {number} perPage  Number of posts per page.
+ * @return {Object}          Object containing Apollo client instance and post archive data or error object.
+ */
+export default async function getPostTypeArchive(
+  postType,
+  orderBy = 'DATE',
+  order = 'DESC',
+  cursor = null,
+  getNext = true,
+  perPage = 10
+) {
+  // Define single post query based on post type.
+  const postTypeQuery = {
+    post: queryPostsArchive
+  }
+
+  // Retrieve post type query.
+  const query = postTypeQuery?.[postType] ?? null
+
+  // Get/create Apollo instance.
+  const apolloClient = initializeApollo()
+
+  // Set up return object.
+  const response = {
+    apolloClient,
+    error: false,
+    errorMessage: null
+  }
+
+  // If no query is set for given post type, return error message.
+  if (!query) {
+    return {
+      apolloClient,
+      error: true,
+      errorMessage: `Post type \`${postType}\` archives are not supported.`
+    }
+  }
+
+  // Determine query variables.
+  const variables = {
```

Comment:
Could this be renamed to something more descriptive like `queryVariables`?

Reply:
@michealengland I went with just `variables` so below we can pass it w/ `query` without having to state the prop and value separately: `.query({query, variables})` but if it's confusing I'm open to renaming.

Reply:
I prefer rename because then it matches the GraphiQL panel a little closer. However, if Apollo typically uses that pattern, I'm okay with that also. I'm going to approve.